packages/devextreme/js/ui/context_menu/ui.context_menu.js

```diff
@@ -556,6 +555,16 @@ class ContextMenu extends MenuBase {
         if(!arg.cancel) {
             this._hideAllShownSubmenus();
             this._setOptionWithoutOptionChange('visible', false);
+
+            const $scrollableElement = this._overlay.$content().find(`.${SCROLLABLE_CLASS}`).first();
```

Comment:
[code splitting]
Looks like a code which should be extracted to a separate method